pkg/services/alerting/notifiers/sensugo_test.go

```diff
@@ -0,0 +1,61 @@
+package notifiers
+
+import (
+	"testing"
+
+	"github.com/grafana/grafana/pkg/components/simplejson"
+	"github.com/grafana/grafana/pkg/models"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func TestSensuGoNotifier(t *testing.T) {
```

Comment:
Please rewrite tests using the standard Go library + testify.

Reply:
The tests were changed from convey to testify.